src/cockpit/superuser.py

```diff
@@ -168,19 +168,19 @@ async def go(self, name: str, responder: ferny.InteractionResponder) -> None:
         except KeyError as exc:
             raise bus.BusError('cockpit.Superuser.Error', f'Unknown superuser bridge type "{name}"') from exc
 
-        self.current = 'init'
+        self.current = name
```

Comment:
The reason we don't do this from the start is because we don't know that it will succeed, and we want the UI to remain in an indeterminate state until we have confirmation one way or the other.

Also: the justification in the commit message is confusing.  Package loading is synchronous and the bridges should be fully configured before `Bridge.__init__` completes and the bridge starts processing messages.  Are we somehow doing an accidental reload on login?